client/src/reducers/controls.js

```diff
@@ -35,6 +35,8 @@ const Controls = (
     categoricalAsBooleansMap: null,
     crossfilter: null,
     dimensionMap: null,
+    userDefinedGenes: [],
```

Comment:
I realize this issue is not new, but I suggest we start a comment on line 23 that documents the purpose/semantics of each field.   Eg, 

/*
  dimensionMap:  object containing all dimensions created on the current crossfilter.  Obs annotations are keyed by annotation name, layout keyed by 'X' and 'Y', and varData cached by var name.
*/